.github/workflows/install_test.yaml

```diff
@@ -37,16 +34,16 @@ jobs:
         with:
           path: ${{ env.pythonLocation }} 
           key: ${{ matrix.os- }}-${{ matrix.python_version }}-install-${{ env.pythonLocation }}-${{ hashFiles('**/pyproject.toml') }}-v01
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+      - name: Set up cmake and libomp
+        run: brew install cmake libomp
```

Comment:
In commit [ce422b5](https://github.com/alteryx/evalml/pull/4423/commits/ce422b5944e1a29541ff2eb673c05ac4f41039cb), the install test for MacOS unilaterally fail because:
>  ERROR: Could not build wheels for lightgbm, which is required to install pyproject.toml-based projects